src/cmap/auth/mongodb_oidc/aws_service_workflow.ts

```diff
@@ -3,6 +3,9 @@ import { readFile } from 'fs/promises';
 import { MongoAWSError } from '../../../error';
 import { ServiceWorkflow } from './service_workflow';
 
+/** Error for when the token is missing in the environment. */
+const TOKEN_MISSING_ERROR = 'AWS_WEB_IDENTITY_TOKEN_FILE must be set in the environment.';
```

Comment:
This and other string constants just a simple refactoring of any non-interpolated string being a constant.